chia-bls/Cargo.toml

```diff
@@ -8,15 +8,22 @@ authors = ["Arvid Norberg <arvid@chia.net>"]
 homepage = "https://github.com/Chia-Network/chia_rs/chia-bls/"
 repository = "https://github.com/Chia-Network/chia_rs/chia-bls/"
 
+[features]
+py-bindings = ["dep:pyo3", "chia_py_streamable_macro", "chia-traits/py-bindings"]
+
 [dependencies]
 chia-traits = { path = "../chia-traits" }
+clvm-traits = { path = "../clvm-traits" }
```

Comment:
These dependencies will need to specify a version, if we're to ever publish these to crates.io